docs/database/database-spans.md

```diff
@@ -268,6 +287,84 @@ Placeholders in a parameterized query SHOULD not be sanitized. E.g. `where id =
 e.g. from `IN (?, ?, ?, ?)` to `IN (?)`, as this can help with extremely long IN-clauses,
 and can help control cardinality for users who choose to (optionally) add `db.query.text` to their metric attributes.
 
+## Synthetic query text
+
+The `db.query.synthetic` attribute captures a shortened representation of a query text
+which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.
+
+It SHOULD only be reported if instrumentation parses query text for the database call.
+
+When query text is parsed, the parser SHOULD extract a list of operations and corresponding
+collection names performed in this query.
+
+The `db.query.synthetic` attribute value SHOULD contain a semicolon separated list
+of operation and corresponding collection names formatted in the following way:
+
+```
+{operation1} {collection1}; {operation2} {collection2}; ...
+````
+
+For example, a
+> [!NOTE]
+> Operations may be performed on an anonymous collection or on multiple collections
+> at the same time, so there could be 0 or more collection names associated with one
+> operation name.
+
+If operation is performed on an anonymous collection, the corresponding collection
+name SHOULD NOT be captured. If operation is performed on multiple collections, collection names
+for this operation should be recorded as comma-separated list.
+
+**Examples**:
+
+- Query that performs single operation:
+
+   ```sql
+   SELECT *
+   FROM   wuser_table
+   WHERE  username = ?
+   ```
+
+   the corresponding `db.query.synthetic` is `SELECT wuser_table`.
+
+- Query that performs multiple operations:
+
+   ```sql
+   INSERT INTO shipping_details
+               (order_id,
+               address)
+   SELECT order_id,
+         address
+   FROM   orders
+   WHERE  order_id = ?
+   ```
+
+   the corresponding `db.query.synthetic` is `INSERT shipping_details; SELECT orders`.
+
+- Query that performs an operation that's applied to multiple collections:
+
+   ```sql
+   SELECT *
+   FROM   songs,
+         artists
+   WHERE  songs.artist_id == artists.id
+   ```
+
+  the corresponding `db.query.synthetic` is `SELECT songs,artists`.
+
+- Query that performs an operation on an anonymous table:
+   ```sql
+   SELECT order_date
+   FROM   (SELECT *
+           FROM   orders o
+                  JOIN customers c
+                    ON o.customer_id = c.customer_id)
+   ```
+
+  the corresponding `db.query.synthetic` is `SELECT; SELECT orders; JOIN orders,customers`.
```

Comment:
and

> SELECT orders customers

Reply:
what about the first `SELECT` ? or should it be `SELECT SELECT orders customers` ? 

also it seems you dropped JOIN - I wanted to clarify that you don't consider it to be an operation, correct?

Reply:
oh, yes, I didn't mean to skip the extra `SELECT` (though potentially we could consider removing consecutive duplicates...)

and yeah, I wasn't considering `JOIN` to be an operation, though it could be...

Reply:
I'd rather not remove duplicates to preserve a bit more of the original query. I added some wording to explain that things like SQL statements is a good start and clauses can be added on case-by-case basis, but there are tradeoffs. PTAL